node/shared/src/snark_job_id.rs

```diff
@@ -53,7 +53,7 @@ impl FromStr for LedgerHashTransitionPasses {
     type Err = ();
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        let (first_pass, second_pass) = s.split_once(":").ok_or(())?;
+        let (first_pass, second_pass) = s.split_once("_").ok_or(())?;
```

Comment:
With dash and underscore it might be harder to read.

Reply:
The same as above